src/Analyzer.Core.BuiltInRuleTests/TestTemplates/RedisCache.badtemplate

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Tabbing for the .badtemplates are different than the previous .badtemplate files

Reply:
Ugh most test files also had inconsistent identations